packages/cosmos-networks/lib/index.ts

```diff
@@ -1,37 +1,106 @@
-import { Network } from '@liquality/types'
+import { Network, cosmos } from '@liquality/types'
 
 export interface CosmosNetwork extends Network {
   network: string
   rpcUrl: string
-  apiUrl: string
-  grpcUrl: string
+  apiUrl?: string
+  grpcUrl?: string
```

Comment:
That's not used anywhere, can we get rid of it?

Reply:
'grpcUrl' will be removed. It's not needed. But 'apiUrl' will be left as optional argument because it might be useful in some cases.